addOns/network/src/main/java/org/zaproxy/addon/network/internal/ContentEncodingsHandler.java

```diff
@@ -40,6 +40,8 @@ public void handle(HttpHeader header, HttpBody body) {
             encodings = List.of(HttpEncodingDeflate.getSingleton());
         } else if (encoding.contains(HttpHeader.GZIP)) {
             encodings = List.of(HttpEncodingGzip.getSingleton());
+        } else if (HttpEncodingBrotli.isAvailable() && encoding.contains("br")) {
```

Comment:
Reported in IRC, one response may be encoded multiple times (ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding), but that can probably be tackled in another PR.

Reply:
Yeah, will fix in other PR.